docs/source/user_input_producer.rst

```diff
@@ -15,31 +15,56 @@ Ensure that the correct URL is present and select the sequence you wish to conne
     * `asynchronously` will send a new document every `x` seconds, taking your modifications on-the-go.
 
-You can configure the maximum number of lines to allow in the subtitle text area by varying the value in the Maximum lines input area. While typing in the subtitle text box pressing Enter will remove lines so that the next document sent does not exceed the Maximum lines setting. Pressing Escape will clear the text area. 
+You can configure the maximum number of lines to allow in the subtitle text area by varying the value in the Maximum lines input area. While typing in the subtitle text box pressing Enter will remove lines so that the next document sent does not exceed the Maximum lines setting. Pressing Escape will clear the text area.
 
 Now you are ready to type subtitle text in the `Subtitle text` box. Each time a new document is sent and received it should be logged by the running consumer script.
 
 .. warning:: Note that the simple configuration above creates a simple consumer that handles the first provided sequence identifier. If the connection is disconnected and a new one made with a different sequence identifier it will reject all documents and close the connection. The workaround to this is to restart the consumer and reconnect.
 
+Handover Manager
+----------------
+
+To see the UIP in action with a handover node, using the Websocket transport mechanism, follow these steps:
+
+* Start the Handover Manager: ``ebu-run --admin.conf=ebu_tt_live/examples/config/user_input_producer_handover.json``
+
+* Create two (or more) instances of the User Input Producer by opening ``ebu_tt_live/ui/user_input_producer/user_input_producer.html`` in multiple browser tabs/windows.
+
+* In each instance of the UIP, create a sequence with a different sequence identifier but with identical authors group identifier. If you use the default configuration, this is "TestGroup1".
+
+* Select the sequence you created and connect. The default configuration publishes to ``ws://127.0.0.1:9001``.
+
+* In the Result host panel, connect to the result host - the default configuration subscribes to ``ws://127.0.0.1:9001``. Enter the Handover Manager's output sequence identifier - the default is "HandoverSequence1".
+
+* Enter a positive number in the 'Authors group control token' field.
+
+* Enter subtitle text and send the document.
+
+* Depending on the control token, the sequence currently selected by the Handover Manager will display an "On air" flag.
+
+* The output from the handover node will appear in the "Received documents" panel. Click a row to reveal the document.
+
+* The output from other UIPs is not output by the Handover Manager until one of then uses a control token higher than the most recently output one.
```

Comment:
This could be replaced with the proposed text below, or if kept, "one of then" -> "one of them".

> * Non-selected sequences can "take control" by sending a document with a higher control token value than the most recently output document from the Handover Manager, according to the EBU-TT Part 3 specification. To show this, enter a higher number in the Control Token field before sending a document.